torch/linalg/__init__.py

```diff
@@ -8,6 +8,51 @@
 # Note: This not only adds doc strings for functions in the linalg namespace, but
 # also connects the torch.linalg Python namespace to the torch._C._linalg builtins.
 
+cholesky = _add_docstr(_linalg.linalg_cholesky, r"""
+linalg.cholesky(input) -> Tensor
+
+Returns the Cholesky decomposition.
```

Comment:
I think combining the first few paragraphs would be helpful. What about something like:

"Computes the Cholesky decomposition of a Hermitian positive-definite matrix or the Cholesky decompositions of a batch of such matrices. If the matrices are real-valued then each of their Cholesky decompositions can be written as A = LL^t, for a lower triangular matrix L, and this function returns the matrix L for each input matrix. If the matrices are complex-valued then their Cholesky decompositions are A = L @ L.H for a lower triangular matrix L, where L.H is the conjugate transpose of L."